source/elements/oneDAL/source/introduction.rst

```diff
@@ -1,3 +1,17 @@
 ============
 Introduction
 ============
```

Comment:
provided the comments and expected representation of the intro in e-mail

Reply:
ok, will keep this in both section. high level intro and subsections with more details probably